meetings/2018-07-06.md

```diff
@@ -0,0 +1,158 @@
+* Dan: I seen that and adding to what Tierney said, it’s often the practitioners that are tasked to sort of get code on 
+  messed up and Promises code tends to exacerbate that situation, the immovable point that I’ve seen around Promises have 
+  come from teams that have gotten to a point of getting enough observe in their systems, through callback wrapping and 
+  various other techniques.
+* Benjamin: Another big part is that the V8 team has been very cooperative and pretty grating like taking feedback we give
+  them seriously and improving things in the engine like async stack traces in production and then the ability to inspect
+  the stock in production, I think a big improvement; now while things are getting better and there are already a lot better 
+  than they were with callbacks at least for the big crowds. I still try to like monitor the Promises tagged in stack 
+  overflow so there are things I still it are getting brought up a lot and we’re asking the V8 team for their time in implementing
+  stuff and they started like implementing different warnings and like things occur often for us which is very of them 
+  to dedicate that time, but I want to make sure that we’re not wasting time and things that like my intuition or like
+  a not personal intuition which is why I think it’s really important that Jordan is here. 
+  The next step is try to figure it out good questions to ask and we would love  your guidance; get the survey out so we can bring
+  it up in the meetings we have with the V8 team.
+* Dan: Ahmad is representing enterprise and has large deployment of node, is our guard Promises a problem, start with this
+  kind of the broadest question.
+* Ahmad: I don’t really hear too much complaints or see kind of situations where like our deployment or our application 
+  development is affected by that, but I think it only becomes a problem in the context of inconsistencies and debugging
+  as we said, it’s not great, but I would echo the point it’s not a burning problem; the nicest thing a developer has to do 
+  in terms of debugging issues when they have to deal with code that’s mixed between Promises and non Promises especially 
+  when it comes down to the native node APIs.
+* Dan: Michael what the current approach TSC is having to dealing with them promised APIs?
+* Michael: At this point its sort of experimental state where we’re at or different people are adding a couple, so far,
+  it’s up to us and DNS it’s sort of experiment and get a feel for what works and what doesn’t as prelude to making more 
+  organized effort to cover the broader surface area. My understanding of the discussion around the debug ability is that 
+  the challenge is, you know, without before Promises if you had an unhandled exception it was easy to track that down because 
+  you could set it up so you got an exception when that happens in the case of Promises because the way I handled rejections
```

Comment:
you could set it up so you got an exception.  When that happens in the case of Promises because the way rejection handlers can be added at any time that's more difficult.